### PR TITLE
samples: Add relationship and per-type bigquery samples to Cloud Asset service

### PR DESCRIPTION
**Pull request by lvvvvvf**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-asset/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

File: samples/snippets/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java [hunk 1]
```diff
@@ -34,21 +36,42 @@
 
   // Export assets for a project.
   // @param args path where the results will be exported to.
-  public static void exportBigQuery(String bigqueryDataset, String bigqueryTable) throws Exception {
+  public static void exportBigQuery(
+      String bigqueryDataset, String bigqueryTable, ContentType contentType, boolean perType)
+      throws Exception {
```
[the remark this answers is not in the document]
Comment (lvvvvvf) [reply samples/snippets/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java]: Our api could return any exceptions like invalid arg, permission deny, out of quota. 
So leave it as it is

File: samples/snippets/src/main/java/com/example/asset/CreateFeedExample.java [hunk 2]
```diff
@@ -27,15 +28,21 @@
...
 public class CreateFeedExample {
   // Create a feed
-  public static void createFeed(String[] assetNames, String feedId, String topic, String projectId)
-      throws Exception {
+  public static void createFeed(
+      String[] assetNames,
+      String feedId,
+      String topic,
+      String projectId,
+      ContentType contentType) throws Exception {
```
Comment (telpirion) [inline samples/snippets/src/main/java/com/example/asset/CreateFeedExample.java:36]: Still need a more precise Exception here.

File: samples/snippets/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java [hunk 3]
```diff
@@ -22,33 +22,58 @@
...
 
-  // Export assets for a project.
-  // @param args path where the results will be exported to.
-  public static void exportBigQuery(String bigqueryDataset, String bigqueryTable) throws Exception {
+  // Export assets to BigQuery for a project.
+  public static void exportBigQuery(
+      String bigqueryDataset,
+      String bigqueryTable,
+      ContentType contentType,
+      boolean isPerType) throws Exception {
```
Comment (telpirion) [inline samples/snippets/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java:42]: Still need a more precise exception here.

File: samples/snippets/src/test/java/com/example/asset/ListAssets.java [hunk 4]
```diff
@@ -64,4 +64,18 @@ public void testListAssetsExample() throws Exception {
       assertThat(got).contains("asset");
     }
   }
+
+  @Test
+  public void testListAssetsRelationshipExample() throws Exception {
```
Comment (telpirion) [inline samples/snippets/src/test/java/com/example/asset/ListAssets.java:69]: Replace with more precise Exception sub-type.

**Review by telpirion — COMMENTED** [review]
Please be sure to "ack" comments that you don't address with a reason why.

File: samples/snippets/src/main/java/com/example/asset/ExportAssetsExample.java [hunk 5]
```diff
@@ -33,10 +34,8 @@
   private static final String projectId = ServiceOptions.getDefaultProjectId();
 
   // Export assets for a project.
-  // @param args path where the results will be exported to.
-  public static void main(String... args) throws Exception {
-    // Gcs path, e.g.: "gs://<my_asset_bucket>/<my_asset_dump_file>"
-    String exportPath = args[0];
+  // @param exportPath where the results will be exported to.
+  public static void exportAssets(String exportPath, ContentType contentType) throws Exception {
```
[the remark this answers is not in the document]
Comment (lvvvvvf) [reply samples/snippets/src/main/java/com/example/asset/ExportAssetsExample.java]: Add in the last commit

File: samples/snippets/src/main/java/com/example/asset/CreateFeedExample.java (hunk 2 above)
Comment (lvvvvvf) [reply samples/snippets/src/main/java/com/example/asset/CreateFeedExample.java]: Add in the last commit

File: samples/snippets/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java (hunk 3 above)
Comment (lvvvvvf) [reply samples/snippets/src/main/java/com/example/asset/ExportAssetsBigqueryExample.java]: Add in the last commit

File: samples/snippets/src/test/java/com/example/asset/ListAssets.java (hunk 4 above)
Comment (lvvvvvf) [reply samples/snippets/src/test/java/com/example/asset/ListAssets.java:69]: Add in the last commit

**Review by telpirion — APPROVED** [review]

**Pull request merged**